js/apps/admin-ui/cypress/e2e/users_test.spec.ts

```diff
@@ -203,7 +203,7 @@ describe("User creation", () => {
 
     cy.wait("@save-user").should(({ request, response }) => {
       expect(response?.statusCode).to.equal(204);
-      expect(request.body.attributes, "response body").deep.equal({
+      expect(request.body.attributes, "response body").deep.contains({
```

Comment:
Using `equals` might end up in false positives because you can have built-in/internal attributes returned as a user attribute.

I watched the video and this assertion was failing because the `locale` attribute is also available in the `attributes` map. 

I'm not sure why it is failing for this specific PR though because the changes here are unrelated to this failure.